src-schema/PhpunitTestCase.php

```diff
@@ -31,6 +31,10 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        if ($this->db !== null) {
```

Comment:
make sure this is never merged, we want to create a new connection for every test case

and it does not solve the connections issue at all...

Reply:
For sure.

It looks like for some reason on Postgre the connection is not dropped after `$this-db = null` in `tearDown`.

Reply:
that is what I said in Discord - you introduced circular references somewhere and refcounting is short, ie. the only way to kick destructors is GC